demos/config.h

```diff
@@ -22,9 +22,37 @@
 #ifndef CONFIG_H
 #define CONFIG_H
 
+
 /* Set network context to socket (int). */
 typedef int MQTTNetworkContext_t;
 
 #define MQTT_MAX_QUEUED_PUBLISH_MESSAGES 10
 
+/*
+ * @brief The Logging Interface Hook.
+ * 
+ * @note Customers can EITHER
+ * Define their port of the logging stack through this macro
+ *          OR
+ * Use the reference POSIX implementation of the logging stack, 
```

Comment:
The comment here should be that this config file maps the logging to a sample POSIX implementation.

Reply:
Updated